setup.py

```diff
@@ -28,15 +28,13 @@ def find_version(*file_paths):
     long_description = f.read()
 
 install_requires = [
+    'enum34  ; python_version < "3.4"',
```

Comment:
Having this marker here will output it in the `opentok.egg-info/requires.txt` file used by proper python package manager.